support-frontend/typescript-errors.json

```diff
@@ -105,8 +103,7 @@
             "assets/components/marketingConsent/marketingConsent.tsx": 1,
             "assets/components/page/page.tsx": 1,
             "assets/components/priceLabel/priceLabel.tsx": 1,
-            "assets/components/subscriptionCheckouts/address/addressFields.tsx": 10,
-            "assets/components/subscriptionCheckouts/address/addressFieldsStore.ts": 12,
+            "assets/components/subscriptionCheckouts/address/addressFields.tsx": 14,
```

Comment:
looks like we might have introduced a few new errors here?

Reply:
I'm going to leave that file for another PR, nothing has changed in that file and you have to stop somewhere with this stuff!